### ISSUE TITLE
Specify "is autosomal" or provide ploidy of a sequence record

### ISSUE THREAD

**Issue by nh13** [samtools/hts-specs#535]
So many tools and groups hard-code or have lists of which contigs are autosomal, or even a specific ploidy (looking at you plant genomes).  I wanted to start a discussion of an optional tag for the sequence records (`SQ` lines) to indicate if the contig is autosomal, or better yet, provide the expected ploidy in the assembly.  The tag may be omitted where not appropriate or known.  This is similar to topology in providing optional information that is **very** useful when present.

I/we am/are guilty of [hard-coding common non-autosomal contigs](https://github.com/fulcrumgenomics/fgbio/blob/57988d615fa6188e0028faedfdf65ed13cf645e7/src/main/scala/com/fulcrumgenomics/util/Sequences.scala#L40) in fgbio.

**Comment by yfarjoun** [comment]
PAR and names of male and female contigs are hard-coded [in Picard as well](https://github.com/broadinstitute/picard/blob/f020c9d0b827cbc5bdb46171d58d2166314630b9/src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java#L170-L177)

Nice idea, however I'm sure there are problems. here's one:

- Contigs may different ploidy in different parts, or for different "types" of individuals:. e.g. chrX and chrY in humans, wrt PAR. 

perhaps we can use this issue to collect possible problems and then look for a solution that resolves them.

**Comment by tfenne** [comment]
I agree @yfarjoun - there are definitely more things that could be handled at the same time.  It would be good to catalog as many as possible and then come up with something that handles many/most of them without trying to solve every last problem.  Here's my list:

- autosome vs. sex chromosome
- ploidy (or perhaps sex-dependent ploidy) in the germline of the species from which a reference comes
- pseudo-autosomal information for sex chromosomes
- alloploidy metadata (e.g. `chrA` is ploidy2 but is related to `chrA1` in the same reference genome)

**Comment by d-cameron** [comment]
circular/linear is already covered in the `@SG` `TP` field.

**Comment by tfenne** [comment]
@d-cameron: @nh13 was saying that this is analogous to the topology field in the header.  I'm pretty sure he's aware of it since he's the one who authored #405 😉
